state/store.go

```diff
@@ -232,12 +239,28 @@ func saveValidatorsInfo(db dbm.DB, height, lastHeightChanged int64, valSet *type
 	valInfo := &ValidatorsInfo{
 		LastHeightChanged: lastHeightChanged,
 	}
-	if lastHeightChanged == height {
-		valInfo.ValidatorSet = valSet
-	}
+	valInfo.ValidatorSet = valSet
```

Comment:
What happened here?

Reply:
I saved validatorSet every block cause we don't have  proposer priority and proposer as @guagualvcha said.

Reply:
I couldnt say it is a good design but I dont know whether there are some better solutions.
We should it really? What result we should bear for this change?